runtime/runtime/src/lib.rs

```diff
@@ -2867,6 +2869,7 @@ mod tests {
     }
 
     #[test]
+    #[ignore] // TODO: figure out why this test stopped passing with #10852, cc @shreyan-gupta
```

Comment:
See the discussion here: https://near.zulipchat.com/#narrow/stream/295302-general/topic/test_storage_proof_size_soft_limit

Reply:
To copy my analysis here:

Oh I think I know: because we no longer fetch the contract wasm from trie upon each execution, the test stops hitting the storage proof limit. Does that sound like a reasonable guess?

If yes, how is statelessnet expected to deal with contract compilation? Are validators expected to know all the contracts for all shards and keep a compiled cache, or should chunk producers supply validators with contract each time? (in which case validators would need quite a bit more processing power than today, because compiling a contract is probably the slowest operation in our system and we currently avoid most of it)

Regardless, I think it makes sense to disable this test just for now because of the hurry to land optimizations in 1.38.1 by EOW, but we should think some more about what exactly this optimization means when considered from the statelessnet perspective.